CSharp/BatchExplorer/Converters/InvertedBooleanConverter.cs

```diff
@@ -12,18 +12,28 @@ namespace Microsoft.Azure.BatchExplorer.Converters
     public class InvertedBooleanConverter : IValueConverter
     {
         public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
-        {
-            if (targetType != typeof (bool))
-            {
-                throw new InvalidOperationException("The target must be a bool.");
-            }
-
-            return !(bool)value;
-        }
+        {
+            return _Convert(value, targetType);
+        }       
 
         public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
```

Comment:
💡 Noticed hence pointing it out :) in both the `Convert` and `ConvertBack` functions, the function signature is exactly same i.e. both take `object value, Type targetType, object parameter, CultureInfo culture` as parameter.

* Idea: Why not get rid of 1 of them because both return ==> `_Convert(value, targetType);` i.e. one of the functs between `Convert` and `CovertBack` seems redundant. :)